### PR TITLE
Use caption component from vue library

### PR DESCRIPTION
**Pull request by marcoambrosini**
Use the upstreamed component.

- [x] https://github.com/nextcloud/nextcloud-vue/pull/1740

### CODE REVIEW

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
linter seems not happy

**Review by nickvergessen — APPROVED** [review]
- [x] Needs vue update

**Comment by PVince81** [comment]
oh I thought the caption component was already in 3.7.0. it's not

File: src/components/RightSidebar/Participants/ParticipantsTab.vue [hunk 1]
```diff
@@ -28,7 +28,7 @@
 			:is-searching="isSearching"
 			@input="handleInput"
 			@abort-search="abortSearch" />
-		<Caption v-if="isSearching && canAdd"
+		<AppNavigationCaption v-if="isSearching && canAdd"
```
Comment (nickvergessen) [inline src/components/RightSidebar/Participants/ParticipantsTab.vue:31]: not imported?

Comment (nickvergessen) [reply src/components/RightSidebar/Participants/ParticipantsTab.vue:31]: Not it's AppNavigation and now you use it in the sidebar... this just shouts for errors popping up in the future

**Pull request merged**